mjit_worker.c

```diff
@@ -938,6 +938,11 @@ compile_compact_jit_code(char* c_file)
     // compacted functions (not done yet).
     bool success = true;
     list_for_each(&active_units.head, child_unit, unode) {
+        if (!IMEMO_TYPE_P(child_unit->iseq, imemo_iseq)) {
+            rp((VALUE)child_unit->iseq);
+            rb_bug("gone");
+        }
+
         char funcname[MAXPATHLEN];
```

Comment:
I'm fine with the rest of the changes, but why the change in mjit_worker.c ? May want to remove this from the PR if it's from upstream.

Reply:
I was just trying to get lucky with these checks on CI. will remove.